File: 8_curso_poo_uber/php/uberX.php
<<<<<<< HEAD
<?php
require_once('car.php')
class uberX extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver)
        $this->brand = $brand;
        $this->model = $model;
    }
}
=======
<?php
require_once('car.php')
class uberX extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver)
        $this->brand = $brand;
        $this->model = $model;
    }
}
>>>>>>> d1bff8aa9e084cc099db80a3377014aa5d90fe12
?>

File: 8_curso_poo_uber/java/UberX.java
<<<<<<< HEAD
class UberX  extends Car{
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
    @Override
    void printDataCar() {
        super.printDataCar();
        System.out.println("Modelo: " + model + " Brand: " + brand);
    }
=======
class UberX  extends Car{
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
    @Override
    void printDataCar() {
        super.printDataCar();
        System.out.println("Modelo: " + model + " Brand: " + brand);
    }
>>>>>>> d1bff8aa9e084cc099db80a3377014aa5d90fe12
}